test.js

```diff
@@ -4,5 +4,13 @@ var fileUrl = require('./');
 
 it('should convert path to file url', function () {
 	assert(/file:\/\/\/.*\/test\.jpg/.test(fileUrl('test.jpg')));
-	assert.strictEqual(fileUrl('/Users/sindresorhus/dev/te^st.jpg'), 'file:///Users/sindresorhus/dev/te%5Est.jpg');
+	if (process.platform === 'win32') {
```

Comment:
This is not really the way to fix it. Some users might have a different drive than `C:`. Instead, just do two check, one that check the it starts with `file:////` and one that it ends in `te%5Est.jpg`.